src/documents/README.md

```diff
@@ -73,21 +89,31 @@ layout: default
     * [UI Options Migration](tutorial-migratingToInfusion1.5/UIOptionsMigration.md)
     * [Pager Migration](tutorial-migratingToInfusion1.5/PagerMigration.md)
 
-## API
+## Tutorials
+* [Getting started with Infusion](tutorial-gettingStartedWithInfusion/GettingStartedWithInfusion.md)
+* Preferences Framework
+  * [Creating a Preferences Editor ](tutorial-prefsFramework/CreatingAPrefsEditor.md)
+  * [Adding a Basic Preferences Editor to a Site](to-do/AddingPrefsEditorToSite.md)
+  * [Working With A Preferences Editor On Your Site](to-do/WorkingWithPrefsEditorOnSite.md)
+  * [Creating a New Adjuster UI](tutorial-prefsFramework/CreatingANewAdjusterUI.md)
+* [User Interface Options](tutorial-userInterfaceOptions/UserInterfaceOptions.md)
+* General
+  * [How to Create and Use Font Icons](tutorial-iconFonts/HowToCreateAndUseFontIcons.md)
+  * [Using String Templates](tutorial-usingStringTemplates/UsingStringTemplates.md)
 
-* Component APIs
-  * [Inline Edit](to-do/InlineEditAPI.md)
-  * [Pager](to-do/PagerAPI.md)
-  * [Progress](to-do/ProgressAPI.md)
-  * [Reorderer](to-do/ReordererAPI.md)
+## Components
+* [Inline Edit](to-do/InlineEditAPI.md)
+* [Pager](to-do/PagerAPI.md)
+* [Progress](to-do/ProgressAPI.md)
+* [Reorderer](to-do/ReordererAPI.md)
     * [List Reorderer](to-do/ListReordererAPI.md)
     * [Grid Reorderer](to-do/GridReordererAPI.md)
     * [Layout Reorderer](to-do/LayoutReordererAPI.md)
     * [Image Reorderer](to-do/ImageReordererAPI.md)
```

Comment:
There is an actual ImageReordererAPI page now, this should be changed to point at that and the todo one removed.